CRM/Member/Form/Search.php

```diff
@@ -192,6 +192,36 @@ protected function getSortNameLabelWithOutEmail() {
     return ts('Member Name');
   }
 
+  /**
+   * Get the label for the tag field.
+   *
+   * We do this in a function so the 'ts' wraps the whole string to allow
+   * better translation.
+   *
+   * @return string
+   */
+  protected function getTagLabel() {
+    return ts('Member Tag(s)');
+  }
+
+  /**
+   * Get the label for the group field.
+   *
+   * @return string
+   */
+  protected function getGroupLabel() {
+    return ts('Member Group(s)');
+  }
+
+  /**
+   * Get the label for the group field.
+   *
+   * @return string
+   */
+  protected function getContactTypeLabel() {
+    return ts('Participant Contact Type');
```

Comment:
@eileenmcnaughton this needs to be changed to member

Reply:
ah yes, opps